src/components/Stake.vue

```diff
@@ -163,6 +163,9 @@ export default defineComponent({
     const resultHash = ref('');
     const referLink = ref('');
 
+    const burnsWarning =
+      'All remaining DOT in that account are burned if the account remains less than 1 DOT';
```

Comment:
Please change the error message to `Account with balance below the existential deposit will be reaped` or something like that according to the polkadot spec